services/galley/schema/src/V37.hs

```diff
@@ -0,0 +1,19 @@
+module V37 (migration) where
+
+import Imports
+import Cassandra.Schema
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 37 "Create table `custom-instances`" $ do
+    schema' [r|
+        CREATE TABLE custom-instances (
+            domain             text,  -- email domain of the custom backend owner.
+            config_json_url    text,  -- url under which this custom backend's
+                                      -- config.json can be got.
+            webapp_welcome_url text,  -- needed for webapp due to CORS trouble
+            block_cloud_users  bool,  -- iff NULL or False, new cloud users
+                                      -- with emails in this domain can be created.
+            PRIMARY KEY (domain)
+        );
+        |]
```

Comment:
@jschaul any opinions on types, names, anything?  should we add any tuning details?

Reply:
https://github.com/wireapp/wire-server/pull/946#discussion_r369003743

this means that the `boolean` here should be an `int`.